styles.css

```diff
@@ -44,6 +45,7 @@ header {
   background: url('img/header-shapes mobile.png');
   width: 462px;
   height: 470px;
+  border-bottom-left-radius: 80px;
 }
```

Comment:
[OPTIONAL] 👍🏽 

- I think the `width`  added  to your `.intro-section` has caused some extra margin on your mobile screen(375px). as this is what I can currently see'

![excessp](https://user-images.githubusercontent.com/99246359/198730002-941425d2-2536-4b80-bae4-069eabf875f0.PNG)

- I think you can fix this excess margin on your  `intro-section` by simply deleting the `width: 462px;` and it will be perfect.

What do you think?? 🤗 🤗